plottable.d.ts

```diff
@@ -2331,6 +2331,51 @@ declare module Plottable {
             boxVisible(show: boolean): SelectionBoxLayer;
             fixedWidth(): boolean;
             fixedHeight(): boolean;
+            /**
+             * Gets the x scale for this SelectionBoxLayer.
+             */
+            xScale<D extends number | {
+                valueOf(): number;
+            }>(): QuantitativeScale<D>;
+            /**
+             * Sets the x scale for this SelectionBoxLayer.
+             *
+             * @returns {SelectionBoxLayer} The calling SelectionBoxLayer.
+             */
+            xScale<D extends number | {
+                valueOf(): number;
+            }>(xScale: QuantitativeScale<D>): SelectionBoxLayer;
+            /**
+             * Gets the y scale for this SelectionBoxLayer.
+             */
+            yScale<D extends number | {
+                valueOf(): number;
+            }>(): QuantitativeScale<D>;
+            /**
+             * Sets the y scale for this SelectionBoxLayer.
+             *
+             * @returns {SelectionBoxLayer} The calling SelectionBoxLayer.
+             */
+            yScale<D extends number | {
+                valueOf(): number;
+            }>(yScale: QuantitativeScale<D>): SelectionBoxLayer;
+            /**
+             * Gets the data values backing the left and right edges of the box.
+             *
+             * Returns an undefined array if the edges are not backed by a scale.
+             */
+            xExtent(): {
+                valueOf(): number;
```

Comment:
Is this going to compile under TS1.4? Right now, I think this will cause a compiler error in Typescript 1.4:
```Typescript
var xExtent = <number[]> selectionBoxLayer.xExtent();
```
because TS1.4 doesn't consider `{ valueOf(): number }` convertible to `number`.